dpdata/amber/md.py

```
@@ -18,6 +18,63 @@
+        1
+        - cos_alpha**2
+        - cos_beta**2
+        - cos_gamma**2
+        + 2 * cos_alpha * cos_beta * cos_gamma
+    )
+    if np.any(z_factor <= 0.0):
+        raise RuntimeError("Invalid AMBER cell angles")
+
+    z = np.sqrt(z_factor) / sin_gamma
+
+    shape = cell_lengths.shape[:-1] + (3, 3)
```

Ruff (0.15.12)

[warning] Consider `(*cell_lengths.shape[:-1], 3, 3)` instead of concatenation

Replace with `(*cell_lengths.shape[:-1], 3, 3)`

(RUF005)